app/templates/macros/page_macros.html

```diff
@@ -7,24 +7,19 @@
        <button class="ui button start-edit">
         Edit
        </button>
-       <p class= "updated_alert">Updated successfully</p>
-       <button class="ui primary button end-edit">
+       <button class="ui primary button end-edit" style="display:none">
```

Comment:
getting this issue when i try to edit after merging in brandon's changes: none of the changes are being saved when "save" is clicked, and i get this error in the terminal: 
` OperationalError: (sqlite3.OperationalError) no such column: suggestions.resource_name [SQL: u'SELECT count(*) AS count_1 \nFROM (SELECT suggestions.id AS suggestions_id, suggestions.resource_id AS suggestions_resource_id, suggestions.suggestion_text AS suggestions_suggestion_text, suggestions.read AS suggestions_read, suggestions.submission_time AS suggestions_submission_time, suggestions.contact_name AS suggestions_contact_name, suggestions.contact_email AS suggestions_contact_email, suggestions.contact_phone_number AS suggestions_contact_phone_number, suggestions.resource_name AS suggestions_resource_name, suggestions.resource_address AS suggestions_resource_address \nFROM suggestions \nWHERE suggestions.read = 0) AS anon_1']
`